### PR TITLE
MaxK Wendy Bite HW

### PR DESCRIPTION
**Pull request by MaxKaye**

### CODE REVIEW

File: MK WendyB Code.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Wendy Bite | My Background</title>
+  <link rel="stylesheet" href="styles.css">
```
Comment (mattscilipoti) [inline MK WendyB Code.md]: I don't see a styles.css file.  Did you have issues moving the css to another file?

**Comment by mattscilipoti** [comment]
We expected you to work on an html file, usually "index.html". I don't see how you can see the results of your efforts unless this is in an html file. Eventually, the css should move out of that file to "styles.css".

**Comment by mattscilipoti** [comment]
:+1: Complete

File: MK WendyB Code.md (hunk 1 above)
Comment (MaxKaye) [inline MK WendyB Code.md]: Yes, thanks for catching that. Initially I tried to upload together but it didn't so I moved CSS into the HTML file. I was able to see the final index.html file when I put it into browser though- were you able to see my version of the webpage?

**Pull request closed without merging**